docs/docs-content/architecture/orchestration-spectrocloud.md

```diff
@@ -56,22 +65,23 @@ Palette's decentralized model is based on a "decentralized management - local po
 
 ![distributed_orchestration.webp](/architecture_orchestartion-spectrocloud_distributed-flow.webp)
 
-As part of the workload K8s cluster provisioning, only the first control-plane node is launched by Cluster API, running
-in the Palette management cluster. Once the control-plane node is operational, Cluster API resources are _pivoted_ from
-the management platform into the target workload cluster.
+As part of the workload Kubernetes cluster provisioning, only the first control-plane node is launched by the Cluster
+API active in the Palette management cluster. Once the control plane node is operational, Cluster API resources are
+_pivoted_ from the Palette management platform into the target workload cluster.
 
-The target workload cluster is then responsible for provisioning and maintaining the remaining control-plane and worker
-nodes. All Day-2 operations which result in node changes, including OS/K8s upgrades, scaling, and K8s certificate
-rotation, are triggered by changes to the Cluster API resources in the target workload cluster.
+The target workload cluster is responsible for provisioning and maintaining the remaining control plane and worker
+nodes. All Day-2 operations, which may result in node changes, including the operating system and Kubernetes upgrades,
+scaling, and Kubernetes SSL certificate rotation, are triggered by changes to Cluster API resources in the target
+workload cluster.
 
-Palette pivots these clusters for several reasons, related to scalability and availability:
+Palette pivots these clusters for reasons such as:
```

Comment:
This sentence is a bit confusing here. We were just talking about the workload cluster, now we are talking about pivot again. I'm not sure if I understood :)

Reply:
@caroldelwing, great catch. Let me take a stab at updating the old language and making it clearer.